war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1310,7 +1310,7 @@ function xor(a,b) {
 // used by editableDescription.jelly to replace the description field with a form
 function replaceDescription() {
     var d = document.getElementById("description");
-    $(d).down().next().innerHTML = "<div class='spinner-right'>loading...</div>";
```

Comment:
Since the description button is not always present, the `.next()` may fail JavaScript when there is no description.